apps/ftu/test/unit/language_test.js

```diff
@@ -101,9 +101,12 @@ suite('languages >', function() {
       assert.isTrue(KeyboardHelper.changeDefaultLayouts.called);
     });
 
-    test('localize changed', function() {
+    test('localize changed', function(done) {
```

Comment:
Nit: this way the test will time out of the assertion fails (throws) (yes I know there are other tests in this file with the same problem). We want to move towards something like:

      return Promise.resolve().then(() => {
        assert.equal(MockNavigatorSettings.mSettings['locale.hour12'], false);
      }).then(done, done);